ui/admin/app/routes/scopes/scope/roles/role/index.js

```diff
@@ -18,7 +18,7 @@ export default class ScopesScopeRolesRoleIndexRoute extends Route {
   async afterModel() {
     const currentScope = this.modelFor('scopes.scope');
     const scopes = await this.store.query('scope', {
-      scope_id: currentScope.id,
+      scope_id: 'global',
```

Comment:
This doesn't fix the underlying issue, if we go to an org scope now this will incorrectly include other org scopes.

<img width="364" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/b761cb7d-f2bf-431f-b644-283d37cae60b">

Reply:
@ZedLi Thanks for the clarification earlier; really helped!